langchain/llms/mlflow.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from typing import Any, List, Dict, Mapping
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+import mlflow.gateway
+
+
+class MlflowGateway(LLM):
+    gateway_uri: str
+    route: str
+    temperature: float = 0.0
+    stop: List[str] | None = None
+    max_tokens: int = 0
+    candidate_count: int = 5
+
+    def __init__(self, **kwargs: Any):
+        import mlflow
+
+        super().__init__(**kwargs)
+        mlflow.gateway.set_gateway_uri(self.gateway_uri)
+
+    @property
+    def _default_params(self) -> Dict[str, Any]:
+        return {
+            "gateway_uri": self.gateway_uri,
+            "route": self.route,
+            "temperature": self.temperature,
```

Comment:
```suggestion
            "temperature": self.temperature,
            ...
```

TODO: Add parameters